client-common/src/main/java/org/apache/cassandra/sidecar/common/response/TableIdsResponse.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.common.response;
+
+import java.util.HashMap;
+
+/**
+ * Simple data type to represent TableIds response
+ *
+ * Format
+ * - Key: Full qualified table name
+ * - Value: UUID string
+ *
+ * Example,
+ * {
+ *   "keyspaceName.tableName1": "tableId1",
+ *   "keyspaceName.tableName2": "tableId2",
+ * }
+ */
+public class TableIdsResponse extends HashMap<String, String>
```

Comment:
Did we agree to remove table id API for now?

Reply:
Removed the class